pkg/stdlib/html/scroll_element.go

```diff
@@ -11,44 +11,85 @@ import (
 
 // SCROLL_ELEMENT scrolls an element on.
 // @param docOrEl (HTMLDocument|HTMLElement) - Target document or element.
-// @param selector (String, options) - If document is passed, this param must represent an element selector.
+// @param selector (String) - If document is passed, this param must represent an element selector.
+// @param options (ScrollOptions) - Scroll options. Optional.
 func ScrollInto(ctx context.Context, args ...core.Value) (core.Value, error) {
-	err := core.ValidateArgs(args, 1, 2)
+	err := core.ValidateArgs(args, 1, 3)
 
 	if err != nil {
 		return values.None, err
 	}
 
-	if len(args) == 2 {
-		doc, err := drivers.ToDocument(args[0])
+	var doc drivers.HTMLDocument
+	var el drivers.HTMLElement
+	var selector values.String
+	var opts drivers.ScrollOptions
+
+	if len(args) == 3 {
+		if err = core.ValidateType(args[1], types.String); err != nil {
+			return values.None, err
```

Comment:
Maybe would be better to wrap errors smth like `errors.Wrap(err, `validate "selector"`)`